src/pages/Team/Team.js

```diff
@@ -259,6 +282,35 @@ class Team extends Component {
       alert("Select a Nest");
     }
   };
-}
 
+  deleteNestUser = (member) => {
+    if (this.state.selectedNest) {
+      const nestID = this.state.selectedNest.nestId;
+      const email = member.email;
+
+      API.graphql(
+        graphqlOperation(mutations.removeNestUser, {
+          nestId: nestID,
+          email: email,
+        })
+      ).then((value) => {
+        const nests = _.cloneDeep(this.state.nests);
+        const updatedNest = {
```

Comment:
Derrick, I'm an idiot with how I told you we can do this. Sorry, I was not thinking. You can change this line to simply be updatedNest = value.data.removeNestUser and then for lines 304 and 307 set those properties just to be updatedNest.